src/content/roadmap/scaling/index.md

```diff
@@ -38,10 +38,10 @@ This second step is known as [“Danksharding”](/roadmap/danksharding/). It is
 
 ## Decentralizing rollups {#decentralizing-rollups}
 
-Rollups are already scaling Ethereum. A rich ecosystem of rollup projects is enabling users to transact quickly and cheaply, with a range of security guarantees. However, rollups have been bootstrapped using centralized sequencers (computers that do all the transaction processing and aggregation before submitting them to Ethereum). This is vulnerable to censorship, because the sequencer operators can be sanctioned, bribed or otherwise compromised. At the same time, [rollups vary](https://l2beat.com) in the way they validate incoming data. The best way is for "provers" to submit fraud proofs or validity proofs, but not all rollups are there yet. Even those rollups that do use validity/fraud proofs use a small pool of known provers. Therefore, the next critical step in scaling Ethereum is to distribute responsibility for running sequencers and provers across more people.
+[Rollups](/layer-2/#rollups) are already scaling Ethereum. A [rich ecosystem of rollup projects](https://l2beat.com/scaling/tvl) is enabling users to transact quickly and cheaply, with a range of security guarantees. However, rollups have been bootstrapped using centralized sequencers (computers that do all the transaction processing and aggregation before submitting them to Ethereum). This is vulnerable to censorship, because the sequencer operators can be sanctioned, bribed or otherwise compromised. At the same time, [rollups vary](https://l2beat.com) in the way they validate incoming data. The best way is for "provers" to submit fraud proofs or validity proofs, but not all rollups are there yet. Even those rollups that do use validity/fraud proofs use a small pool of known provers. Therefore, the next critical step in scaling Ethereum is to distribute responsibility for running sequencers and provers across more people.
```

Comment:
Noting here that the `#rollups` hash link doesn't work... the h3 element used on this page does not have an id to link to. The link still takes you to the Layer 2 page, but will stay at the top.

Separate from this PR, but ideally we have these on all h2/h3 elements throughout the entire site. For this one specifically, we could just add `id="rollups"` to the `h3` element in [layer-2.tsx line 603](https://github.com/ethereum/ethereum-org-website/blob/bcd22b190b9853e04e31aec5cd15a8a650867dfd/src/pages/layer-2.tsx#L603).